src/storage/activities.tsx

```diff
@@ -160,9 +188,33 @@ export const useActivities = (): [Activities, ModifyActivities] => {
     return false;
   };
 
+  /**
+   * Removes an entry from the specified activity list.
+   * 
+   * @remarks: This function ignores all matching instances and does not check
+   *  if they connect or not.
+   *
+   * @param entry - The entry to be removed
+   * @returns A new activity list with the entry removed
+   */
+  const remove = (activity: ActivitiesEntry, day: ActivitiesDay, activities: Activities): Activities => {
+    const dayIndex = activities.findIndex(elem => elem === day);
+    if(dayIndex !== -1) {
+      // Clear out any instance of the given acitivity
+      const newEntries: (ActivitiesEntry | null)[] = [];
+      activities[dayIndex].entries.forEach(elem => {
+        newEntries.push(elem ? (activityEntryEq(elem, activity) ? null : elem) : null);
+      });
+      // Update entry list
+      activities[dayIndex].entries = newEntries;
+    }
+    return [ ...activities ];
+  };
+
   const modifyActivities: ModifyActivities = {
     add: add,
     addInterval: addInterval,
+    remove: (activity, day) => setStoreItem(activitiesKey, remove(activity, day, activities)),
```

Comment:
Unless remove is used somewhere else setStoreItem should be moved inside the function similar to the others.

Reply:
Got it, i was wondering about this when i wrote it. Since my knowledge about the inner workings of the storage module is limited i mainly took inspiration from how the calendar functions worked, thus its similar look. But of course this could be moved in to a separate function to match the current files structure. Will do.